src/main/java/com/commercetools/project/sync/inventoryentry/InventoryEntrySyncer.java

```diff
@@ -0,0 +1,72 @@
+package com.commercetools.project.sync.inventoryentry;
+
+import static com.commercetools.sync.inventories.utils.InventoryReferenceReplacementUtils.replaceInventoriesReferenceIdsWithKeys;
+
+import com.commercetools.project.sync.Syncer;
+import com.commercetools.sync.inventories.InventorySync;
+import com.commercetools.sync.inventories.InventorySyncOptions;
+import com.commercetools.sync.inventories.InventorySyncOptionsBuilder;
+import com.commercetools.sync.inventories.helpers.InventorySyncStatistics;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.expansion.ExpansionPath;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.expansion.InventoryEntryExpansionModel;
+import io.sphere.sdk.inventory.queries.InventoryEntryQuery;
+import java.util.List;
+import javax.annotation.Nonnull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class InventoryEntrySyncer
+    extends Syncer<
+        InventoryEntry,
+        InventoryEntryDraft,
+        InventorySyncStatistics,
+        InventorySyncOptions,
+        InventoryEntryQuery,
+        InventorySync> {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(InventoryEntrySyncer.class);
+
+  /** Instantiates a {@link Syncer} instance. */
+  private InventoryEntrySyncer(
+      @Nonnull final InventorySync inventorySync,
+      @Nonnull final InventoryEntryQuery query,
```

Comment:
```suggestion
      @Nonnull final InventoryEntryQuery inventoryQuery,
```

Reply:
isn't it seen from the type :) ?

Reply:
For consistency :) You have this kind of name in the other syncers as well.

Reply:
@lojzatran true: 66b2bf5